File: Recursivos/Contador/Contador.c
#include <stdio.h>

void Imprimir(int n){
    if(n == 1){
        printf("%d ", n);
    }else{
        Imprimir(n - 1);
        printf("%d ", n);
    }
}

int main(){
    int n;
    scanf("%d", &n);
    Imprimir(n);

    return 0;
}

File: Recursivos/Contador/Contador.java
import java.util.Scanner;

public class Contador {

    public static void imprimir(int n){
        if(n == 1){
            System.out.print(n + " ");
        }else{
            imprimir(n-1);
            System.out.print(n + " ");
        }
    }
    public static void main(String[] args){
        try(Scanner scanner = new Scanner(System.in)){
            int n = scanner.nextInt();
            imprimir(n);

            scanner.close();
        }
    }
}
